action.yml

```diff
@@ -29,6 +29,13 @@ inputs:
       Override the detected FOSSA project name. If running FOSSA analysis on a
       Pull Request, as a start you can use the contexts `github.ref` or `github.ref_name`.
     required: false
+  team:
+    description: >-
```

Comment:
Should we note that these only work when you are creating a project? As noted here: https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-yml.md#project

I think it would be nice, but we don't document this when you run `fossa analyze --help`, so it's obviously not 100% necessary

Reply:
resolved in caefed67